src/Microsoft.Azure.SignalR.Common/RestClients/HttpExceptionWrapper.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using Microsoft.Rest;
+
+namespace Microsoft.Azure.SignalR.Common.RestClients
+{
+    public class HttpExceptionWrapper
```

Comment:
make it a static exception extension class?

Reply:
usage: 
wrapper.WrapException(Exception e) -->
e.ConvertToAzureSignalRException()

@vicancy waht do you think?

Reply:
I think this advice is good. Already done.